tests/test_file_utils.py

```
@@ -0,0 +1,1078 @@
+        builtin_config = server_root / "conf" / "custom_models.json"
+
+        # Test with the actual builtin path
+        result = _is_builtin_custom_models_config(str(builtin_config))
+        # Result depends on whether the file actually exists
+        assert isinstance(result, bool)
+
+    def test_is_builtin_custom_models_config_false(self):
+        """Test non-builtin config file returns False"""
+        with tempfile.NamedTemporaryFile(suffix=".json") as f:
+            result = _is_builtin_custom_models_config(f.name)
+            assert result == False
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not result:` for false checks

Replace with `not result`

(E712)